post-deploy-playbooks/post-deploy-ssm-processing.yml

```diff
@@ -57,6 +57,7 @@
         - { name: Cron nats_house_keeping, user: wherescape, minute: "*/45",  hour: "*", day: "*", job: sh -c ". /etc/profile.d/script_envs.sh && /NATS/scripts/nats_house_keeping.sh > /dev/null 2> /dev/null" }
         - { name: Cron ftp_acl_web02.py, user: wherescape, minute: "*/10",  hour: "0", day: "*",  job: sh -c ". /etc/profile.d/script_envs.sh && cd /ADT/scripts && python ftp_acl_web02.py" }
         - { name: Cron sftp_oag_client_maytech.py, user: wherescape, minute: "*/2",  hour: "*", day: "*",  job: sh -c  ". /etc/profile.d/script_envs.sh && cd /ADT/scripts && python sftp_oag_client_maytech.py > /dev/null 2> /dev/null" }
+        - { name: Cron oag_backload_data_move.sh, user: root, minute: "*/2",  hour: "*", day: "*",  job: /usr/bin/oag_backload_data_move.sh }
```

Comment:
Don't you need ` sh -c` for the job?

Reply:
No, because the job is one command running under root